config/jobs/kubernetes-client/haskell/client-haskell-presubmits.yml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-client/haskell:
+  - name: client-haskell-unit-tests
+    always_run: true
+    decorate: true
```

Comment:
I've found it a good practice to include `optional: true` to presubmits.

In that way, the test will run, but if there's a glitch with the test that causes it to fail, people can still merge code.
After it's run a few times, then it's okay to remove the `optional:true` line.

Reply:
Makes sense, made the change!